openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -56,17 +56,22 @@ This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiabl
 
 The following aspects are in scope of this interoperability profile:
 
-* Protocol for issuance of the Verifiable Credentials (can be both remote and in-person) (OID4VCI)
-* Protocol for online presentation of Verifiable Credentials (can be both remote and in-person) (OID4VP)
+* Profile of OpenID4VCI to issue IETF SD-JWT VCs (can be both remote and in-person), including
```

Comment:
Why mention "(can be both remote and in-person)" on this line, but not with the ones below?

Reply:
```suggestion
* Profile of OpenID4VCI to issue IETF SD-JWT VCs, including
```
I think we can delete it.